README.md

```diff
@@ -12,12 +12,25 @@ Our templates offer a rich boilerplate to jump start Go Gin-based application de
 
 ## Get Started
 
-## Usage
+### Prerequisite 
+- [Python interpreter](https://docs.python.org/3/using/index.html)
+- Cookiecutter
+  - Mac: `brew install cookiecutter`
+  - Ubuntu: `brew install cookiecutter`
+- Go version >= 1.15
 
-Clone the repository
+## Usage
+- Download **latest** version of gin-templates.
+  ```
+  go get github.com/nimblehq/gin-templates
+  ```
 
-`git clone git@github.com:nimblehq/git-template.git`
+- Run create the project command.
+  ```
+  gin-templates create
+  ```
 
+- Follow the instruction and the app should be created.
```

Comment:
Line break needed and rephrasing needed:
```suggestion
- Follow the instructions in the terminal.
- You new application is created 🎉 .

```

Reply:
Fixed in abd9be910833e2fd834ea6f9a0349a1cbbd4b5ba